pkg/controller/namespacelabel/namespacelabel_controller.go

```diff
@@ -53,9 +78,9 @@ func Add(mgr manager.Manager) error {
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 	ctx, cancel := context.WithCancel(context.Background())
-	operatorNs := "redhat-rhmi-operator"
+	operatorNs := namespace
```

Comment:
It looks like we load the namespace later in the reconcile function. `ns, err := GetNS(r.context, r.operatorNamespace, r.client)` Is it necessary to add it to the struct? Thinking there might be some unintended consequences having this hardcoded?

Reply:
good point, I'll update it.